src/main/java/com/scottlogic/filters/combine/OrCombineFilters.java

```diff
@@ -0,0 +1,36 @@
+package com.scottlogic.filters.combine;
+
+import com.scottlogic.PostFilter;
+import com.scottlogic.UserPost;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+public class OrCombineFilters implements PostFilter {
+    final private PostFilter filter1;
+    final private PostFilter filter2;
+
+    public OrCombineFilters(PostFilter inputFilter1, PostFilter inputFilter2){
+        filter1 = inputFilter1;
+        filter2 = inputFilter2;
+    }
+
+
+    @Override
+    public List<UserPost> filter(List<UserPost> inputList) {
+        if(inputList == null){return null;}
+        List<UserPost> outputList2 = filter1.filter(inputList);
+        List<UserPost> outputList3 = filter2.filter(inputList);
+
+        List<UserPost> outputList = new ArrayList<>(outputList2);
+        outputList.addAll(outputList3);
+
+        final Set<UserPost> s = new LinkedHashSet<>(outputList);
+        outputList.clear();
+        outputList.addAll(s);
```

Comment:
Not a bad way of doing this, at the moment any duplicate posts that existed in either list will also get filtered out which might not be exactly what you want. It is a rare case though but might be an interesting exercise to see if you can use your `And` filter to avoid this issue.